s3/replication/replicator/src/s3replicator/app.py

```diff
@@ -17,11 +17,95 @@
 # please email opensource@seagate.com or cortx-questions@seagate.com.
 #
 
-from s3replicationcommon.log import Logger
+from aiohttp import web
+import logging
+from .config import Config
```

Comment:
s/.config/config

Reply:
As this file is inside the package itself, it can be referred with either relative path (**as in the commit**) 
or 
starting from the package itself :- **from s3replicationmanager.config import Config**
I can not directly call  'config' as it is not in the path.

Reply:
Ok, lets retain what you have.